README.md

```diff
@@ -9,26 +9,21 @@
-|                            | 📄 Full-Text Search       | Comprehensive text search across any language, powered by tsvector.                        |
-| **Data Handling**          | ✔ Complete SQL Support    | Full SQL support, enabling joins and filters without limitations or extra configuration.   |
-|                            | 🔗 Async indexing         | Non-blocking inserts with up-to-date query readiness.                                      |
-|                            | 🔄 Easy Data Management   | No need for syncing vectors and metadata with external vector DB, simplifying development. |
-| **Data Types**             | 🔢 FP16/INT8 Data type    | Supports FP16 and INT8 data types for improved storage and computational efficiency.       |
-|                            | 🌓 Binary vector support  | Vector indexing with binary vectors, and Jaccard distance support.                         |
-|                            | 🔪 Matryoshka embeddings  | Subvector indexing, like vector[0:256], for enhanced Matryoshka embeddings.                |
-|                            | ⬆️ Extended Vector Length | Vector lengths up to 65535 supported, ideal for the latest cutting-edge models.            |
-| **System Performance**     | 🚀 Production Ready       | Battle-tested database ecosystem integrated with PostgreSQL.                               |
-|                            | ⚙️ High Availability      | Logical replication support to ensure high availability.                                    |
-|                            | 💡 Resource Efficient     | Efficient attribute storage leveraging PostgreSQL.                                         |
-| **Security & Permissions** | 🔒 Permission Control     | Easy access control like read-only roles, powered by PostgreSQL.                           |
+pgvecto.rs is a Postgres extension that provides vector similarity search functions. It is written in Rust and based on [pgrx](https://github.com/tcdi/pgrx).
+
+## Comparison with pgvector
+
+Checkout [pgvecto.rs vs pgvector](https://docs.pgvecto.rs/faqs/comparison-pgvector.html) for more details.
+
+| Feature | pgvecto.rs | pgvector |
+| --- | --- | --- |
+| Filtering | Introduces VBASE method for vector search and relational query (e.g. Single-Vector TopK + Filter + Join). | When filters are applied, the results may be incomplete. For example, if you originally intended to limit the results to 10, you might end up with only 5 results with filters. |
+| Sparse Vector Search | Supports both dense and sparse vector search. | Supports dense vector search. |
+| Vector Dimensions | Supports up to 65535 dimensions. | Supports up to 2000 dimensions. |
+| SIMD | SIMD instructions are dynamically dispatched at runtime to maximize performance based on the capabilities of the specific machine. | Relies on compiler-generated SIMD code at compile time. |
```

Comment:
pgvector added "Added CPU dispatching for distance functions on Linux x86-64" in 0.7.0.